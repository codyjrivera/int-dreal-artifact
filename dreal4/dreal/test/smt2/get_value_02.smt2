(set-logic QF_NRA)
(set-option :smtlib2_compliant true)

(declare-fun x () Real)
(declare-fun y () Real)
(declare-fun z () Real)
(declare-fun w () Real)
(declare-fun i () Int)
(declare-fun j () Int)
(declare-fun b1 () Bool)
(declare-fun b2 () Bool)

(assert (< 3 x))
(assert (< x 4))
(assert (= (sin x) 0))
(assert (= (sqrt 2) y))
(assert (= z (/ (* 3 4) 2)))
(assert (= w (- x)))
(assert (= i (/ (* 3 4) 4)))
(assert (= j (- i)))
(assert (= b1 (and (> x y)
		   (> y z))))
(assert (= b2 (or (> x y)
		  (> y z))))

(check-sat)
(get-value (x y z w i j b1 b2))

(exit)
