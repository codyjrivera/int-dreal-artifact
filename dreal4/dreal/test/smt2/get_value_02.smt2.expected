delta-sat
(
	(x (interval (closed (/ 884279719003555 281474976710656)) (closed (/ 7074237752028441 2251799813685248))))
	(y (interval (closed (/ 1592262918131443 1125899906842624)) (closed (/ 3184525836262887 2251799813685248))))
	(z 6)
	(w (interval (closed (/ (- 7074237752028441) 2251799813685248)) (closed (/ (- 884279719003555) 281474976710656))))
	(i 3)
	(j (- 3))
	(b1 false)
	(b2 true)
)
